frontend/src/Components/Table/Cells/VirtualTableRowCell.js

```diff
@@ -19,11 +21,13 @@ function VirtualTableRowCell(props) {
 
 VirtualTableRowCell.propTypes = {
   className: PropTypes.string.isRequired,
-  children: PropTypes.oneOfType([PropTypes.string, PropTypes.node])
+  children: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+  title: PropTypes.bool
```

Comment:
title should be a string not a bool. Children could be a node in which case you are setting title to the node. Just pass in a string and use it